### PR TITLE
Localizable strings

### PR DESCRIPTION
**Pull request by SerhiiShovkoplias**
## Description
all NSLocalizedString aggregated to Localizable.strings

## Motivation and Context
tari-project/wallet-ios#107

## How Has This Been Tested?
run tests, manual testing

## Types of changes

* [ ] UI fix (non-breaking change which fixes a UI issue)
* [ ] Functionality bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [x] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [x] I have tested this on multiple simulator devices with different screensizes.
* [x] I'm merging against the `development` branch.
* [x] Commits have been squashed into one commit with a descriptive commit message
* [x] I ran all tests before pushing.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added/changed tests to cover my changes if not UI changes.

### CODE REVIEW

File: MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift [hunk 1]
```diff
@@ -64,15 +64,15 @@ class ReminderNotifications {
...
+            body: NSLocalizedString("short-body-scheduled-reminder", comment: "Local reminder notifications"),
             deliverAfter: 60 * 60 * 24
         ),
         ScheduledReminder(
             identifier: "scheduled-reminders-recipient-2",
             title: ReminderNotifications.titleString,
             body: String(
                 format: NSLocalizedString(
-                    "Someone sent you %@! Open Tari Aurora to receive it now or it will be returned to the sender.",
+                    "long-body-scheduled-reminder %@",
```
Comment (Jasonvdb) [inline MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:75]: You can remove the %@ from the key here. That's a placeholder var, it only needs to be in the value string

Comment (SerhiiShovkoplias) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:75]: i know, this is to make it clear that there is a variable

Comment (Jasonvdb) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:75]: Ok cool 👍

File: MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift [hunk 2]
```diff
@@ -54,7 +54,7 @@ class ReminderNotifications {
 
     static let titleString = String(
         format: NSLocalizedString(
-            "You've been sent %@!",
+            "scheduled-reminder-title %@",
```
Comment (kukabi) [inline MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:57]: You should parameters inside the string resource, not here. The location of the parameters could be different for different languages.

Comment (SerhiiShovkoplias) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:57]: it's just stub to make it clear that there is a variable.

Comment (kukabi) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:57]: I see, saw the convo about this after I made this comment. @Jasonvdb what do you think?

**Comment by kukabi** [comment]
In general regarding the namings, I think it's good to go from general to specific, so how about something like:

`{short_view_name}_{context_name_if_any}_{string_resource_name}_{any_other_suffix_regarding_function}`

An example is `icloud_backup_zip_error_description` instead of `failedToCreateZip-error-description`.

And for common strings that are used by more than 1 view:

`common_{string_resource_name}_{any_other_suffix_regarding_function}`.

@SerhiiShovkoplias @Jasonvdb

**Comment by kukabi** [comment]
If a string resource can be grouped (such as a dialog title and description) you may use the suffix to differentiate, such as `icloud_backup_zip_error_description` and `icloud_backup_zip_error_title`. But if there's no such grouping you don't need to have the final suffix that describes the function, so if the zip error example is not appearing in a dialog context you can go for just `icloud_backup_zip_error`. @SerhiiShovkoplias @Jasonvdb

File: MobileWallet/Backup/ICloudBackup.swift [hunk 3]
```diff
@@ -47,26 +47,23 @@ enum ICloudBackupWalletError: Error {
...
     case unableCreateBackupFolder
-    case privateKeyError
 }
 
 extension ICloudBackupWalletError: LocalizedError {
     public var errorDescription: String? {
         switch self {
         case .failedToCreateZip:
-            return NSLocalizedString("Failed to create wallet backup archive", comment: "backup archive error description")
+            return NSLocalizedString("failedToCreateZip-error-description", comment: "backup archive error description")
```
Comment (kukabi) [inline MobileWallet/Backup/ICloudBackup.swift:56]: `icloud_backup_zip_error`

File: MobileWallet/Backup/ICloudBackup.swift [hunk 4]
```diff
@@ -47,26 +47,23 @@ enum ICloudBackupWalletError: Error {
...
         case .noICloudBackupExists:
-            return NSLocalizedString("You don't have any wallets backed up with iCloud", comment: "'No any wallet backup' error description")
+            return NSLocalizedString("noICloudBackupExists-error-description", comment: "'No any wallet backup' error description")
```
Comment (kukabi) [inline MobileWallet/Backup/ICloudBackup.swift:58]: `icloud_backup_no_backup_exists_error`

File: MobileWallet/Backup/ICloudBackup.swift [hunk 5]
```diff
@@ -47,26 +47,23 @@ enum ICloudBackupWalletError: Error {
...
         case .unarchiveError:
-            return NSLocalizedString("Unable to unarchive wallet from iCloud backup", comment: "unarchive wallet error description")
+            return NSLocalizedString("unarchiveError-description", comment: "unarchive wallet error description")
```
Comment (kukabi) [inline MobileWallet/Backup/ICloudBackup.swift:60]: `icloud_backup_unzip_error`

File: MobileWallet/Backup/ICloudBackup.swift [hunk 6]
```diff
@@ -47,26 +47,23 @@ enum ICloudBackupWalletError: Error {
...
         case .dbFileNotFound:
-            return NSLocalizedString("Unable to create wallet backup. File not found", comment: "sqlite file not found error description")
+            return NSLocalizedString("dbFileNotFound-error-description", comment: "sqlite file not found error description")
```
Comment (kukabi) [inline MobileWallet/Backup/ICloudBackup.swift:62]: `icloud_backup_file_not_found_error`

File: MobileWallet/Backup/ICloudBackup.swift [hunk 7]
```diff
@@ -47,26 +47,23 @@ enum ICloudBackupWalletError: Error {
...
         case .iCloudContainerNotFound:
-            return NSLocalizedString("Can't connect to iCloud Drive services. Make sure that you are signed in with your Apple ID on this device and that iCloud is enabled.", comment: "iCloud container not found error description")
+            return NSLocalizedString("iCloud-container-not-found-error-description", comment: "iCloud container not found error description")
```
Comment (kukabi) [inline MobileWallet/Backup/ICloudBackup.swift:64]: `icloud_backup_container_not_found_error`

File: MobileWallet/Backup/ICloudBackup.swift [hunk 8]
```diff
@@ -47,26 +47,23 @@ enum ICloudBackupWalletError: Error {
...
         case .unableCreateBackupFolder:
-            return NSLocalizedString("Unable to create backup folder", comment: "Unable to create backup folder error description")
-        case .privateKeyError:
-            return NSLocalizedString("Unable to restore wallet. Private key error", comment: "Unable to restore wallet private key error descroption")
+            return NSLocalizedString("unableCreateBackupFolder-error-description", comment: "Unable to create backup folder error description")
```
Comment (kukabi) [inline MobileWallet/Backup/ICloudBackup.swift:66]: icloud_backup_folder_error

File: MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift [hunk 9]
```diff
@@ -64,15 +64,15 @@ class ReminderNotifications {
         ScheduledReminder(
             identifier: "scheduled-reminders-recipient-1",
             title: ReminderNotifications.titleString,
-            body: NSLocalizedString("Open Tari Aurora", comment: "Local reminder notifications"),
+            body: NSLocalizedString("short-body-scheduled-reminder", comment: "Local reminder notifications"),
```
Comment (kukabi) [inline MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:67]: `scheduled_reminder_short_body`

File: MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift (hunk 1 above)
Comment (kukabi) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:75]: `scheduled_reminder_body_long`

Comment (SerhiiShovkoplias) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:75]: @kukabi without "%@"?

File: MobileWallet/Common/CopyableLabel.swift [hunk 10]
```diff
@@ -130,7 +130,7 @@ extension CopyableLabel {
         greenView.backgroundColor = Theme.shared.colors.transactionCellValuePositiveText?.withAlphaComponent(0.12)
 
         let copiedLabel = UILabel()
-        copiedLabel.text = NSLocalizedString("COPIED", comment: "Emoji ID copied")
+        copiedLabel.text = NSLocalizedString("emoji-copied-action-title", comment: "Emoji ID copied")
```
Comment (kukabi) [inline MobileWallet/Common/CopyableLabel.swift:133]: `emoji_id_copied` - it looks generic but if you have something similar in the future you can make it more specific. Shorter is better imo if it's possible to get away with it.

File: MobileWallet/Common/DeepLinkManager.swift [hunk 11]
```diff
@@ -113,8 +113,8 @@ class DeeplinkNavigator {
                     homeVC.onSend(pubKey: pubKey, deepLinkParams: params)
                 } catch {
                     UserFeedback.shared.error(
-                        title: NSLocalizedString("Invalid link found", comment: "Deep link error"),
-                        description: NSLocalizedString("Could not get emoji ID from link", comment: "Deep link error"),
+                        title: NSLocalizedString("emoji-from-deep-link-error-title", comment: "Deep link error"),
```
Comment (kukabi) [inline MobileWallet/Common/DeepLinkManager.swift:116]: `deep_link_from_emoji_id_error_title`

File: MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift (hunk 1 above)
Comment (SerhiiShovkoplias) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:75]: What we decided about %@? It's have not influence to string but helps to visual mark localizable string with variable.

File: MobileWallet/Common/DeepLinkManager.swift [hunk 12]
```diff
@@ -113,8 +113,8 @@ class DeeplinkNavigator {
...
+                        description: NSLocalizedString("emoji-from-deep-link-error-description", comment: "Deep link error"),
```
Comment (kukabi) [inline MobileWallet/Common/DeepLinkManager.swift:117]: `deep_link_from_emoji_id_error_description`

File: MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift (hunk 1 above)
Comment (kukabi) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:75]: Let's wait for @Jasonvdb 's comment after I finish my review. @SerhiiShovkoplias

File: MobileWallet/Common/DeepLinkManager.swift [hunk 13]
```diff
@@ -140,7 +140,7 @@ class ShortcutParser {
         let showQRShortcutItem = UIApplicationShortcutItem(
             type: ShortcutKey.showQR.rawValue,
             localizedTitle: NSLocalizedString(
-                "Show my QR",
+                "show-qr-shortcut",
```
Comment (kukabi) [inline MobileWallet/Common/DeepLinkManager.swift:143]: `shortcut_show_my_qr`

File: MobileWallet/Common/DeepLinkManager.swift [hunk 14]
```diff
@@ -152,7 +152,7 @@ class ShortcutParser {
             type: ShortcutKey.send.rawValue,
             localizedTitle: String(
                 format: NSLocalizedString(
-                    "Send %@",
+                    "send %@",
```
Comment (kukabi) [inline MobileWallet/Common/DeepLinkManager.swift:155]: `shortcut_send`

File: MobileWallet/Common/Extensions/Date.swift [hunk 15]
```diff
@@ -47,9 +47,9 @@ extension Date {
     */
     func relativeDayFromToday() -> String? {
         if Calendar.current.isDateInToday(self) {
-            return NSLocalizedString("Today", comment: "Transaction list section heading")
+            return NSLocalizedString("transaction-list-today-header-title", comment: "Transaction list section heading")
```
Comment (kukabi) [inline MobileWallet/Common/Extensions/Date.swift:50]: `home_tx_list_today`

File: MobileWallet/Common/Extensions/Date.swift [hunk 16]
```diff
@@ -47,9 +47,9 @@ extension Date {
...
         } else if Calendar.current.isDateInYesterday(self) {
-            return NSLocalizedString("Yesterday", comment: "Transaction list section heading")
+            return NSLocalizedString("transaction-list-yesterday-header-title", comment: "Transaction list section heading")
```
Comment (kukabi) [inline MobileWallet/Common/Extensions/Date.swift:52]: `home_tx_list_yesterday` or ``home_transaction_list_yesterday``

File: MobileWallet/Common/Extensions/LAContext.swift [hunk 17]
```diff
@@ -54,8 +54,8 @@ extension LAContext {
 
         var rawValue: String {
             switch self {
-            case .logIn: return NSLocalizedString("Log in to your account", comment: "Authentication")
-            case .userVerification: return NSLocalizedString("User Verification", comment: "Authentication")
+            case .logIn: return NSLocalizedString("logIn-authenticate-user-reason", comment: "Authentication")
```
Comment (kukabi) [inline MobileWallet/Common/Extensions/LAContext.swift:57]: `authenticate_reason_login`

File: MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift (hunk 2 above)
Comment (kukabi) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:57]: `scheduled_reminder.title`

File: MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift (hunk 1 above)
Comment (kukabi) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:75]: `scheduled_reminder.body.long`

File: MobileWallet/Common/Extensions/LAContext.swift [hunk 18]
```diff
@@ -54,8 +54,8 @@ extension LAContext {
...
+            case .userVerification: return NSLocalizedString("userVerification-authenticate-user-reason", comment: "Authentication")
```
Comment (kukabi) [inline MobileWallet/Common/Extensions/LAContext.swift:58]: `authenticate_reason.verification`

File: MobileWallet/Common/Extensions/LAContext.swift [hunk 19]
```diff
@@ -94,7 +94,7 @@ extension LAContext {
                     if success {
                         onSuccess()
                     } else {
-                        let localizedReason = error?.localizedDescription ?? NSLocalizedString("Failed to authenticate", comment: "Failed Face/Touch ID alert")
+                        let localizedReason = error?.localizedDescription ?? NSLocalizedString("authenticate-user-error", comment: "Failed Face/Touch ID alert")
```
Comment (kukabi) [inline MobileWallet/Common/Extensions/LAContext.swift:97]: `authenticate.error.failure`

File: MobileWallet/Common/Extensions/LAContext.swift [hunk 20]
```diff
@@ -104,16 +104,16 @@ extension LAContext {
                 }
             }
         case .none:
-            let alert = UIAlertController(title: NSLocalizedString("Authentication Error", comment: "No biometric or passcode") ,
-                                          message: NSLocalizedString("Tari Aurora was not able to authenticate you. Do you still want to proceed?", comment: "No biometric or passcode"),
+            let alert = UIAlertController(title: NSLocalizedString("authentication-user-error-title", comment: "No biometric or passcode") ,
```
Comment (kukabi) [inline MobileWallet/Common/Extensions/LAContext.swift:107]: `authenticate.error.no_authentication.title`

File: MobileWallet/Common/Extensions/LAContext.swift [hunk 21]
```diff
@@ -104,16 +104,16 @@ extension LAContext {
...
+                                          message: NSLocalizedString("authentication-error-no biometric-description", comment: "No biometric or passcode"),
```
Comment (kukabi) [inline MobileWallet/Common/Extensions/LAContext.swift:108]: `authenticate.error.no_authentication.description`

File: MobileWallet/Common/Extensions/LAContext.swift [hunk 22]
```diff
@@ -104,16 +104,16 @@ extension LAContext {
...
                                           preferredStyle: .alert)
-            alert.addAction(UIAlertAction(title: NSLocalizedString("Try again", comment: "Try again button"),
+            alert.addAction(UIAlertAction(title: NSLocalizedString("try-again-button-title", comment: "Try again button"),
```
Comment (kukabi) [inline MobileWallet/Common/Extensions/LAContext.swift:110]: `common.try_again`

File: MobileWallet/Common/Extensions/LAContext.swift [hunk 23]
```diff
@@ -104,16 +104,16 @@ extension LAContext {
...
                                           style: .cancel,
                                           handler: { [weak self] _ in
                                             self?.authenticateUser(onSuccess: onSuccess)
             }))
 
-            alert.addAction(UIAlertAction(title: NSLocalizedString("Proceed", comment: "Proceed button"), style: .default, handler: { _ in
+            alert.addAction(UIAlertAction(title: NSLocalizedString("proceed-button-title", comment: "Proceed button"), style: .default, handler: { _ in
```
Comment (kukabi) [inline MobileWallet/Common/Extensions/LAContext.swift:116]: `common.proceed`

File: MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift (hunk 2 above)
Comment (Jasonvdb) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:57]: It does look a little weird having that in the key. But I understand what @SerhiiShovkoplias means here. Your call, I don't feel too strongly either way

Comment (kukabi) [reply MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift:57]: I see the point too, I agree, I don't feel strongly either. If it'll be explicit having a suffix like `.with_param` might read better. `scheduled_reminder.title.with_param`.

**Pull request merged**